packages/testing/package.json

```diff
@@ -39,11 +39,13 @@
     "@open-wc/chai-dom-equals": "^0.12.36",
     "@open-wc/semantic-dom-diff": "^0.19.3",
     "@open-wc/testing-helpers": "^2.0.0-next.0",
+    "chai-a11y-axe": "^1.3.1",
+    "mocha": "^8.0.0"
+  },
+  "devDependencies": {
     "@types/chai": "^4.2.11",
     "@types/chai-dom": "^0.0.9",
     "@types/mocha": "^5.2.7",
-    "@types/sinon-chai": "^3.2.3",
-    "chai-a11y-axe": "^1.3.1",
-    "mocha": "^6.2.2"
+    "@types/sinon-chai": "^3.2.3"
```

Comment:
why do you think it should be moved to dev deps? it's used by this package, so users get the correct types

Reply:
Fair... I am going to revert this change